### ISSUE TITLE
Rust and Cargo version being used is not displayed

### ISSUE THREAD

**Issue by ltfschoen** [rust-lang/rust-playpen#76]
How does a user of the rust-playpen know what version of the Rust Compiler and Cargo is currently being used in the rust-playpen (i.e. is it based on Nightly or Alpha or something else)?

There does not appear to be a terminal available for users to input ```rustc --version``` and ```cargo --version``` in order to check.

I am currently using the following on my local machine, but occasionally when I get compiler error messages I like to copy and paste the erroneous code onto rust-playpen to compare the error messages before I publish questions in the [Rust online forum](http://users.rust-lang.org/) (so far they are pretty much the same except when trying to use stdin):
rustc 1.0.0-nightly (270a677d4 2015-03-07) (built 2015-03-07)
cargo 0.0.1-pre-nightly (dd7c7bd 2015-03-04) (built 2015-03-04)

**Comment by cjwelborn** [comment]
Someone else can answer this better than me, but here is a [playpen example](http://is.gd/T2GFuv) that will print `rustc`'s release string, commit hash, commit date, and build date. Currently '`1.0.0-nightly`' built on '`2015-03-29`'.

**Comment by crumblingstatue** [comment]
I agree that it would be very useful to know which version of rustc the playpen uses.
For example, if you want to reproduce rustc bugs on it, you would like to know the version it uses.

Are there any technical barriers to getting the rustc version when it's updated and storing it somewhere and exposing it to the clients?

One way to expose them would be to display the versions in the tooltips when you hover over the Stable/Beta/Nightly buttons in the channel selection section.

**Comment by alexcrichton** [comment]
@crumblingstatue there's no technical blocker, just some work that needs to be done!
